### ISSUE TITLE
error: Logitech receiver not found

### ISSUE THREAD

**Issue by oblitum** [pwr-Solaar/Solaar#352]
As [requested](https://github.com/pwr/Solaar/issues/137#issuecomment-289174490):

```
~ ❯❯❯ solaar --debug
22:38:57,154     INFO [MainThread] root: language en_US (UTF-8), translations path None
22:38:57,218     INFO [MainThread] solaar.upower: connected to system dbus, watching for suspend/resume events
22:38:57,241     INFO [MainThread] solaar.ui.notify: starting desktop notifications
22:38:57,265     INFO [MainThread] solaar.listener: starting receiver listening threads

(solaar:21184): Gdk-CRITICAL **: gdk_window_thaw_toplevel_updates: assertion 'window->update_and_descendants_freeze_count > 0' failed
^C
~ ❯❯❯ solaar show
solaar: error: Logitech receiver not found
```

Using a [MK220 combo](http://www.logitech.com/en-roeu/product/wireless-combo-mk220) (in truth for my country it's a K220 which is almost identical combo).  
Installed [solaar-git](https://aur.archlinux.org/packages/solaar-git).

**Comment by oblitum** [comment]
I won't be using this keyboard for too long. I got a Anne PRO which is on its way, so, just reporting for sake of it, hoping to help meanwhile it's still of any use.

**Comment by rvelhote** [comment]
Hello,

I have this keyboard + mouse combo as well and I am having the same issue. Using the git version as well I get the same `receiver not found` error.

Here is the output of `lsusb` regarding the receiver line

`Bus 001 Device 004: ID 046d:c534 Logitech, Inc. Unifying Receiver`

What can I do to help solve the issue with this device?

Thank you,
Ricardo

**Comment by jrbenito** [comment]
This will be solved by PR #337

**Comment by pfps** [comment]
There have been several changes to solaar over the past couple of years so this issue may have already been addressed.

If you are still using solaar and this issue has not been resolved please download and run the current version from this github repository. If the problem still persists add a new comment here, providing the output of `solaar -V` and `solaar show` and any new information about the issue.

If you are still using solaar you should be able to download and run its current version (in Linux) as follows:

1. Install git on your machine if it is not already installed. This is generally done via `sudo apt install git` or `sudo dnf install git`
2. Change to a directory where you want the code.
3. Download solaar via `git clone https://github.com/pwr-Solaar/Solaar.git`
4. Run solaar via `Solaar/bin/solaar`

For more information on installing solaar from github see https://github.com/pwr-Solaar/Solaar/blob/master/docs/installation.md

**Comment by pfps** [comment]
This should have been fixed by PR #337.  If it wasn't, please reopen this issue.
